api-inference-community/docker_images/superb/app/pipelines/speech_classification.py

```diff
@@ -0,0 +1,57 @@
+import os
```

Comment:
Few things
* Please add `speech-classification` to `common` as well.
* Could we also add a test with Lewis repo?

Reply:
Sure !

Reply:
Done.